phpunit/tests/fonts/font-library/wpFontLibrary/getFontCollections.php

```diff
@@ -12,33 +12,40 @@
  */
 class Tests_Fonts_WpFontLibrary_GetFontCollections extends WP_UnitTestCase {
 
-	public static function set_up_before_class() {
-		$font_library = new WP_Font_Library();
+	public function set_up() {
+		// Resets the private static property WP_Font_Library::$collections to empty array.
+		$reflection = new ReflectionClass( 'WP_Font_Library' );
+		$property   = $reflection->getProperty( 'collections' );
+		$property->setAccessible( true );
+		$property->setValue( array() );
+	}
+
+	public function tear_down() {
+		// Resets the private static property WP_Font_Library::$collections to empty array.
+		$reflection = new ReflectionClass( 'WP_Font_Library' );
+		$property   = $reflection->getProperty( 'collections' );
+		$property->setAccessible( true );
+		$property->setValue( array() );
+	}
```

Comment:
I consolidated those lines as a the reset_collection() method to avoid the repetition.

I think set_up() and tear_down() are preferred in core instead of @before and @after.